pkg/util/k8sutil/k8sutil.go

```diff
@@ -297,44 +298,9 @@ func MakeEtcdPod(m *etcdutil.Member, initialCluster []string, clusterName, state
 		},
 		Spec: api.PodSpec{
 			Containers: []api.Container{
-				{
-					// TODO: fix "sleep 5".
-					// Without waiting some time, there is highly probable flakes in network setup.
-					Command: []string{"/bin/sh", "-c", fmt.Sprintf("sleep 5; %s", commands)},
-					Name:    "etcd",
-					Image:   MakeEtcdImage(cs.Version),
-					Ports: []api.ContainerPort{
-						{
-							Name:          "server",
-							ContainerPort: int32(2380),
-							Protocol:      api.ProtocolTCP,
-						},
-					},
-					// a pod is alive only if a get succeeds
-					// the etcd pod should die if liveness probing fails.
-					LivenessProbe: &api.Probe{
-						Handler: api.Handler{
-							Exec: &api.ExecAction{
-								Command: []string{"/bin/sh", "-c",
-									"ETCDCTL_API=3 etcdctl get foo"},
-							},
-						},
-						InitialDelaySeconds: 10,
-						TimeoutSeconds:      10,
-						// probe every 60 seconds
-						PeriodSeconds: 60,
-						// failed for 3 minutes
-						FailureThreshold: 3,
-					},
-					VolumeMounts: []api.VolumeMount{
-						{Name: "etcd-data", MountPath: etcdDir},
-					},
-				},
+				etcdContainer(commands, cs.Version),
```

Comment:
this is not a refactor. now we have two functions that share the same code block. we need to avoid the unnecessary duplication.